aquality-automation/aquality-tracking-ui
# Feature/aquality tracking ui issue 26

# PR Details

Fix issuie with Chart clicks on Test Run View

## Related Issue
#26  

## How Has This Been Tested

1. Go to Test Run View
1. Click Result or resolution charts section
1. Make Sure results are filtered

## Checklist
